lambdas/journeyengine/src/main/java/uk/gov/di/ipv/core/journeyengine/JourneyEngineHandler.java

```diff
@@ -217,8 +215,8 @@ private JourneyEngineResult executeJourneyEvent(
         }
     }
 
-    private void validateJourneyStep(JourneyStep journeyStep) throws JourneyEngineException {
-        Arrays.stream(JourneyStep.values())
+    private JourneyStep getJourneyStep(String journeyStep) throws JourneyEngineException {
+        return Arrays.stream(JourneyStep.values())
```

Comment:
Ah right, this makes sense now!

Reply:
Yeah, good suggestion Chris, makes managing the JourneySteps easier (rather than using constants).